extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -77,6 +77,30 @@ namespace Microsoft.Windows.DevHome.SDK
         Windows.Foundation.IAsyncOperation<ProviderOperationResult> CloneRepositoryAsync(IRepository repository, String cloneDestination, IDeveloperId developerId);
     };
 
+    [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 2)] interface IRepositoryProvider2
+    requires IRepositoryProvider
```

Comment:
nit: indentation for this line.

Reply:
I'm going to leave this as is.  VS does not have a style guide for IDLs and uses a "Default" indentation guide.  I'll leave it how VS wants it until Vs has the correct indentation for IDLs.